docs/remote-docs.sh

```diff
@@ -80,7 +80,7 @@ function html_fn() {
         local link=$(sed -e 's?.so man1/\(.*\)?\1?' <$dir/links/${file%.md})
         markdown=$dir/$link.md
     fi
-    pandoc --ascii --standalone \
+    pandoc --ascii --standalone --from markdown-smart \
```

Comment:
pandoc's syntax is a little confusing here, this means --from markdown *minus* smart